packages/go_router/example/lib/main.dart

```diff
@@ -5,6 +5,15 @@
 import 'package:flutter/material.dart';
 import 'package:go_router/go_router.dart';
 
+// This scenario demonstrates a simple two-page app.
+//
+// The first route '/' is mapped to Page1Screen, and the second route '/page2'
+// is mapped to Page2Screen. To navigate between pages, press the buttons on the
+// pages.
+//
+// The onPress callbacks use context.go() to navigate to another page. This is
+// equivalent to entering url to the browser url bar directly.
```

Comment:
I'm not sure how common it is to use two top-level routes like this, and if it's the best sample to start with. The transition animation looks buggy to me. Maybe we should change the configuration to be a parent and child route to demonstrate the stacking instead:

```dart
  final GoRouter _router = GoRouter(
    routes: [
      GoRoute(
        path: '/',
        builder: (context, state) => const HomeScreen(),
        routes: [
          GoRoute(
            path: 'recents',
            builder: (context, state) => const RecentlyPlayedScreen(),
          ),
        ],
      ),
    ],
  );
```